libc-bottom-half/cloudlibc/src/libc/fcntl/openat.c

```diff
@@ -11,156 +11,61 @@
 #include <fcntl.h>
 #include <string.h>
 
-#ifdef __wasilibc_unmodified_upstream // fstat
-static_assert(O_APPEND == __WASI_FDFLAG_APPEND, "Value mismatch");
-static_assert(O_DSYNC == __WASI_FDFLAG_DSYNC, "Value mismatch");
-static_assert(O_NONBLOCK == __WASI_FDFLAG_NONBLOCK, "Value mismatch");
-static_assert(O_RSYNC == __WASI_FDFLAG_RSYNC, "Value mismatch");
-static_assert(O_SYNC == __WASI_FDFLAG_SYNC, "Value mismatch");
-#else
```

Comment:
There removals of the ifdef blocks seem unrelated?    Could they land separately?